dialog/file.go

```diff
@@ -644,6 +648,13 @@ func showFile(file *FileDialog) *fileDialog {
 
 	d.setLocation(file.effectiveStartingDir())
 	d.win.Show()
+	if file.save {
+		// Focus on the filename Entry as soon as the dialog is shown
+		go func() {
+			time.Sleep(25 * time.Millisecond)
+			file.dialog.win.Canvas.Focus(file.dialog.fileName.(*widget.Entry))
+		}()
```

Comment:
You can focus things before they are shown as long as the items are part of the window contents. Spinning up a goroutine and sleeping for a random amount of time seems like a hacky workaround.

Reply:
Done. I see why.

Reply:
Remove the comment as well. It no longer valid (technically wasn't 100% valid before either).

Reply:
Done